catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -268,6 +288,27 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
         this.attributeOverrides = attributeOverrideMap;
     }
 
+    private String getBlackListedFilePatternString() {
```

Comment:
Renamed the method to `getBlackListAsRegex` to be more clear what this is doing.